lib/__tests__/team.test.ts

```
@@ -351,4 +352,25 @@ describe('team utilities', () => {
       expect(initials).toBe('MW')
     })
   })
+
+  describe('getTeamMemberIds', () => {
+    it('should return an array of strings', () => {
+      const ids = getTeamMemberIds()
+
+      expect(Array.isArray(ids)).toBe(true)
+      ids.forEach(id => expect(typeof id).toBe('string'))
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)